src/plugins/GamepadManager.js

```diff
@@ -351,7 +361,10 @@ module.exports = class GamepadManager {
      * @param {number} remoteIndex index of this gamepad in the remote VM
      */
     listenForInputs(localIndex, remoteIndex) {
```

Comment:
I guess you could rename these to `hostIndex` and `guestIndex` for clarity & consistency with the desktop app